### ISSUE TITLE
Does burn have a converter from PTH file

### ISSUE THREAD

**Issue by longzou** [tracel-ai/burn#1463]
Mostly, there are many models saved by pytorch. So, does the burn have the converter to help this?

**Comment by laggui** [comment]
Yes! We have `PyTorchFileRecorder` (see [reference in the book](https://burn.dev/book/import/pytorch-model.html)).

Since PyTorch saves a (flattened) state dictionary without any model structure definition, you need to define the model in Burn to import the weights you want to reuse.

*Shameless plug:* if you're looking to port your model over to Burn, take a look at [this tutorial](https://dev.to/laggui/transitioning-from-pytorch-to-burn-45m) to port the `torchvision` ResNet weights to the Burn implementation.
